api_tests/registrations/views/test_registration_forks.py

```diff
@@ -19,269 +16,327 @@
-        assert_equal(len(forked_logs), len(expected_logs))
+        assert set(expected_logs) == set(log['attributes']['action'] for log in forked_logs)
+        assert len(forked_logs) == len(expected_logs)
 
         forked_from = data['embeds']['forked_from']['data']
-        assert_equal(forked_from['id'], self.private_registration._id)
+        assert forked_from['id'] == private_registration._id
+
+    #   test_authenticated_non_contributor_cannot_access_private_registration_forks_list
+        non_contributor = AuthUserFactory()
 
-    def test_authenticated_non_contributor_cannot_access_private_registration_forks_list(self):
-        res = self.app.get(self.private_registration_url, auth=self.user_two.auth, expect_errors=True)
-        assert_equal(res.status_code, 403)
-        assert_equal(res.json['errors'][0]['detail'], 'You do not have permission to perform this action.')
+        res = app.get(private_registration_url, auth=non_contributor.auth, expect_errors=True)
+        assert res.status_code == 403
+        assert res.json['errors'][0]['detail'] == 'You do not have permission to perform this action.'
 
+@pytest.mark.django_db
+class TestRegistrationForkCreate:
 
-class TestRegistrationForkCreate(ApiTestCase):
+    @pytest.fixture()
+    def user_two(self):
+        return AuthUserFactory()
 
-    def setUp(self):
-        super(TestRegistrationForkCreate, self).setUp()
-        self.user = AuthUserFactory()
-        self.user_two = AuthUserFactory()
-        self.user_three = AuthUserFactory()
-        self.private_project = ProjectFactory(creator=self.user)
+    @pytest.fixture()
+    def user_three(self):
+        return AuthUserFactory()
 
-        private_pointer = ProjectFactory(creator=self.user_two)
-        actual_pointer = self.private_project.add_pointer(private_pointer, auth=Auth(self.user_two), save=True)
+    @pytest.fixture()
+    def private_pointer(self, user_two):
+        return ProjectFactory(creator=user_two)
 
-        self.private_registration = RegistrationFactory(creator=self.user, project=self.private_project)
+    @pytest.fixture()
+    def private_project(self, user, user_two, private_pointer):
+        private_project = ProjectFactory(creator=user)
+        private_project.add_pointer(private_pointer, auth=Auth(user_two), save=True)
+        return private_project
 
-        self.fork_data = {
+    @pytest.fixture()
+    def private_registration(self, user, private_project):
+        return RegistrationFactory(creator=user, project=private_project)
+
+    @pytest.fixture()
+    def fork_data(self):
+        return {
             'data': {
                 'type': 'nodes'
             }
         }
 
-        self.fork_data_with_title = {
+    @pytest.fixture()
+    def fork_data_with_title(self):
+        return {
             'data': {
                 'type': 'nodes',
                 'attributes':
                     {'title': 'My Forked Project'}
```

Comment:
The above expands out all {} pairs and this one has a single line with {} wrapping title.